code/src/components/app-table/components/app-table-sort-label/AppTableSortLabel.types.ts

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from "react";
+
+import { TableSortLabelProps } from "@mui/material/TableSortLabel";
+
+import { SortOrder } from "@/types/common";
+
+export type AppTableSortLabelProps = Omit<
+  TableSortLabelProps,
+  "active" | "direction" | "onClick"
+> & {
+  sortDirection: SortOrder;
+  active?: boolean;
+  onClick?: () => void;
```

Comment:
do we really need children and active in this type? It's not included in type TableSortLabelProps?

Reply:
Removed